### ISSUE TITLE
Add Referrer as field for queues

### ISSUE THREAD

**Issue by mj-fox** [osTicket/osTicket#4859]
I really like the new referrer feature. However, unless someone tells you that a ticket was referred to you, you can't find it in your tickets and that your attention for that ticket is requested. I wanted to create a queue for my agents "Referred to me" to give them easy and quick access to a all the tickets that were referred to them. Unfortunately Referrer is not available as a field for queue creation, yet.

I suggest adding Referrer to the available fields for queue generation so you can filter for "Referrer includes me/agent".

**Comment by JediKev** [comment]
@mj-fox 

Just saying that referred tickets appears in your `Open` queues. I do agree that we need a filter-able queue field for referrals though; would make it a lot easier and you could separate it into its own queue.

Cheers.

**Comment by mj-fox** [comment]
Yes, I already found them under Open but I for example, have four departments I look over an it's not apparent which of the hundred tickets are explicitly referred to me and require my attention, and which tickets are just regular tickets someone else is taking care of. Being able to separate them into an own queue, would be awesome.
